packages/xod-tethering-inet/src/AtInternet.rei

```diff
@@ -0,0 +1,48 @@
+// A main file that exposes the interface to work with the package.
+
+open Belt;
+
+// :: [(Link, Net)]
+type openConnections = List.t((Connection.Link.t, Net.t));
+
+// :: (Link, Bytes to send, Bytes sent)
+type sending = (Connection.Link.t, int, int);
+
+// State
+type t = {
+  mux: ref(bool),
+  sending: ref(sending),
+  connections: ref(openConnections),
+  events: EventEmitter.t,
+};
+
+// Creates an object with a default state
+let _getDefaultState: unit => t;
```

Comment:
It starts with an underscore. Is it private? If so, remove from `.rei`, leave in `.re` only. If not, remove the underscore.